### DISCUSSION TITLE
在新域名“绳网.top”中脚本失效

### DISCUSSION THREAD

**Discussion by match-fire** [share121/inter-knot#1471] [其他]
解决方案：
将脚本第七行“// @match [https://xn--5o0anh.top/*”](https://xn--5o0anh.top/*%E2%80%9D)
换成// @match https://www.xn--5o0anh.top/*
报错原因：粗心少了个www
@share121
@VacuolePaoo

**Comment by GanlinSama** [comment]
感谢大佬，原来是这样我说怎么突然进不去了

**Comment by VacuolePaoo** [comment]
这个问 @share121 咯，虽然是我解析的，但是脚本作者是他

**Comment by capacitor1** [comment]
match：莫名其妙被艾特了doge
